public/api.php

```diff
@@ -25,13 +25,17 @@
 $DB = new DB_MYSQL;
 $Debug = new DEBUG;
 $Twig = new Twig\Environment(
-    new \Twig\Loader\FilesystemLoader(__DIR__ . '/../templates'),
+    new \Twig\Loader\FilesystemLoader([
+        __DIR__ . '/../templates',
+        __DIR__ . '/../src/locales',
+    ]),
     [
         'debug' => CONFIG['DEBUG_MODE'],
         'cache' => __DIR__ . '/../cache/twig'
```

Comment:
could be '/../.cache/twig'

Reply:
These are the existing code mate. No changes. I added locales to the filesystem loader so that the locales can be accessed from within the api.